### DISCUSSION TITLE
Can I set plugins.release_vcs.git.allow_no_upstream in rezconfig.py?

### DISCUSSION THREAD

**Discussion by JerryKon** [AcademySoftwareFoundation/rez#1639] [Q&A] [answered]
Hello,
I know we can set it in the ./rezplugins/release_vcs/rezconfig. But dont want change the installed code. Can I config it in rezconfig.py or some other config file out of rez package? or how to set it in the repo's package.py?
I tried updating following code in rezconfig.py but not working.

```
plugins = {
          "release_vcs":{
          "git":{"allow_no_upstream":True},
}
```

**Comment by predat** [comment, marked as answer]
Hi @JerryKon,

you can put this into `~/.rezconfig.py` or use `REZ_CONFIG_FILE` env var to point to a specific config file:
```python
plugins = {
    "release_vcs": {
        "git": {
            "allow_no_upstream": True
        }
    }
}
```
You can use the `rez-config` command to check that the parameter has been taken into account:
```bash
$ rez-config plugins.release_vcs.git
allow_no_upstream: true
```
